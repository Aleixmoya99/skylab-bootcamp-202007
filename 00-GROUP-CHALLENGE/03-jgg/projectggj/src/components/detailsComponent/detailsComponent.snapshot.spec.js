import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import DetailsComponent from './detailsComponent';
const props = {
    match: {
        params: {
            name
        }
    }
}

describe('DetailsComponent snapShot', () => {
    const DetailsComponentTree = renderer.create(
        <Router>
            <DetailsComponent {...props} />
        </Router>
    );
    it('should macth', () => {
        expect(DetailsComponentTree).toMatchSnapshot();
    })
});