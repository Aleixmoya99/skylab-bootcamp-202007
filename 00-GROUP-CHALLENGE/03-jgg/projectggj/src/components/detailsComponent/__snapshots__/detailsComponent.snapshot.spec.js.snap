// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsComponent snapShot should macth 1`] = `
<div
  className="details-holder"
>
  <div
    className="flex-item"
  >
    <div
      className="row"
    >
      <div
        className="detail-container flex-item"
      >
        <div
          className="card hovercard"
        >
          <div
            className="cardheader"
          />
          <div
            className="avatar"
          />
          <div
            className="info"
          >
            <div
              className="title"
            >
              <a
                href="/"
                onClick={[Function]}
                target="_blank"
              />
            </div>
            <div
              className="details-desc-content"
            >
              <div
                className="details-desc"
              >
                RACE: 
              </div>
              <div
                className="details-desc"
              >
                STATUS: 
              </div>
              <div
                className="details-desc"
              >
                PLANET: 
              </div>
              <div
                className="details-desc"
              >
                GENDER: 
              </div>
              <div
                className="details-desc"
              >
                SERIE: 
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
